tibber_aws/aws_logs.py

```diff
@@ -0,0 +1,149 @@
+import logging
+from dataclasses import dataclass
+from datetime import datetime as dt
+import json
+from .aws_base import AwsBase
+from typing import Dict, List
+from types_aiobotocore_logs.client import CloudWatchLogsClient
+
+logger = logging.getLogger("aws_logs")
+
+
+@dataclass
+class CloudWatchLogEvent:
+    """An AWS CloudWatch log event (a log line)"""
+    logStreamName: str
+    timestamp: int
+    message: str
+    ingestionTime: int
+    eventId: str
+
+    def __post_init__(self):
+        self.timestamp_dt = dt.fromtimestamp(self.timestamp / 1000)
+        self.ingestionTime_dt = dt.fromtimestamp(self.ingestionTime / 1000)
+        self.message = json.loads(self.message)
```

Comment:
This will fail for any log message that isn't JSON? At this level, arguably, we should keep a schema that maps to what the message schema is IMO, so we can read any cloudwatch logs, not just those that are JSON

Reply:
could maybe have `.json()` as a helpful property? Eg. like how a lot of web frameworks let you ask for request bodies as JSON, but don't assume the body will *always* be JSON